### PR TITLE
GML

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#40 
#### What does this implement/fix? Explain your changes.
- GML format added
- `AUTHORS.md` updated

### CODE REVIEW

File: AUTHORS.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Core Developers #
 
 ----------
-- Sepand Haghighi - Sharif University Of Technology - ([@sepandhaghighi](http://github.com/sepandhaghighi)) ([sepand@qpage.ir](mailto:sepand@qpage.ir))
\ No newline at end of file
+- Sepand Haghighi - Sharif University Of Technology - ([@sepandhaghighi](http://github.com/sepandhaghighi)) ([sepand@pyrgg.ir](mailto:sepand@pyrgg.ir))
+- Sadra Sabouri - Sharif University Of Technology - ([@sadrasabouri](https://github.com/sadrasabouri)) ([sadra@pyrgg.ir](mailto:sadra@pyrgg.ir))
```
Comment (sadrasabouri) [inline AUTHORS.md:5]: Thank you :smile:

File: README.md [hunk 2]
```diff
@@ -395,6 +397,53 @@ or send an email to [info@pyrgg.ir](mailto:info@pyrgg.ir "info@pyrgg.ir").
        node_n,node_2,weight_n 
 	```
 
+- GML(.gml)
+	```
+	   graph
+	   [
```
Comment (sadrasabouri) [inline README.md:407]: It seems this line is not aligned with line 444 `README.md`

File: README.md [hunk 3]
```diff
@@ -395,6 +397,53 @@ or send an email to [info@pyrgg.ir](mailto:info@pyrgg.ir "info@pyrgg.ir").
...
+         ]
+         node
+         [
+          id 2
+          label "Node 2"
+         ]
+         .
+         .
+         .
+         [
```
Comment (sadrasabouri) [inline README.md:423]: It'd better if we add `node` here, like:
```
- [
+ node
+ [
```

Comment (sepandhaghighi) [reply README.md]: Thanks, fixed in https://github.com/sepandhaghighi/pyrgg/pull/46/commits/19c53dc883ca6b2e09474c8ed305a0449586589e

**Comment by sepandhaghighi** [comment]
Thanks for your review ;-)                                                                                                                                                                                                                                                                                                                                        Sent from my BlackBerry 10 smartphone.

**Pull request merged**